tests/datastructures/test_headers.py

```diff
@@ -271,10 +270,10 @@ def test_cache_control_from_header_single_value() -> None:
     header_value = "no-cache"
     header = CacheControlHeader.from_header(header_value)
     header_dict = header.dict(exclude_unset=True, exclude_none=True, by_alias=True)
-    assert header_dict == {"no-cache": True}
+    assert header_dict == {"documentation-only": False, "no-cache": True}
 
 
-@pytest.mark.parametrize("invalid_value", ["x=y=z", "x, ", "no-cache=10"])
```

Comment:
Same as above, is this being removed because it was incorrect? I'd imagine that we'd want the new implementation to pass all the same tests as the previous, unless there was something fundamentally wrong with the test. In which case an explanation would be useful.